smartsim/_core/control/controller.py

```diff
@@ -618,7 +618,7 @@ def reload_saved_db(self, checkpoint_file: str) -> Orchestrator:
             if not self._jobs.actively_monitoring:
                 self._jobs.start()
 
-            return orc
+            return orc # type: ignore # noqa: no-any-return
```

Comment:
instead of the `# type: ignore`/`# noqa: ` comments here, can we instead to an explicit cast at ~line 603?

Reply:
🤦 I don't know what I was thinking. Done!